css/at-rules/font-face.json

```diff
@@ -43,6 +43,186 @@
             "deprecated": false
           }
         },
+        "OpenType_CBDT_CBLC": {
+          "__compat": {
+            "description": "OpenType CBDT and CBLC rendering",
+            "spec_url": [
```

Comment:
Feel free to link these specs from the content, but in BCD they don't qualify as a valid spec that we can lint with browser-specs etc.

" Each URL must link to a specification published by a standards body or a formal proposal that may lead to such publication." -- this is not the case here. It also doesn't look like our typical exception in https://github.com/mdn/browser-compat-data/blob/main/test/linter/test-spec-urls.ts.

I think it is not a big deal to not have them given we have no mdn_url for these features either.

Reply:
Removed!